boxoffice/views/admin_item_collection.py

```diff
@@ -50,8 +51,11 @@ def jsonify_item_collection(item_collection_dict):
 def admin_item_collection(item_collection):
     item_ids = [str(item.id) for item in item_collection.items]
     date_item_counts = counts_per_date_per_item(item_collection, g.user.timezone)
-    date_sales = sales_by_date(date_item_counts.keys(), g.user.timezone, item_ids)
-    today_sales = date_sales.get(datetime.datetime.now().strftime("%Y-%m-%d"), Decimal(0)) if date_sales else Decimal(0)
+    date_sales = {}
+    for date_stamp in date_item_counts.keys():
+        sales_datetime = datetime.datetime.strptime(date_stamp, '%Y-%m-%d')
```

Comment:
The function providing this data is defined in the models. There's no reason for it to be formatting as string only to have it parsed back as date in the view.